app/javascript/scavenger_hunt/scenes/PlayArena/MainContent/MainContent.jsx

```diff
@@ -1,77 +1,32 @@
-import React, { useState } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import './MainContent.scss';
 import { Clues } from '../../../data/staticData/clues';
-import {
-  Grid,
-  InputLabel,
-  TextField,
-  FormHelperText
-} from '@material-ui/core';
-import { Button } from 'react-bootstrap';
+import { 
+    QuestionOne,
+    QuestionTwo,
+    QuestionThree
+  } from '../../Questions';
+import { Grid } from '@material-ui/core';
 
 
 const MainContent = ({ progress, clueId }) => {
-  const [answer, setAnswer] = useState("");
-
   const getClue = () => {
     let index = clueId - 1
     if ((clueId - 1) > progress) {
       window.location.href = `/hunt/play/${progress + 1}`
       index = progress
     }
-    
-    return Clues[index]
-  }
 
-  const renderQuestion = () => {
-    const clue = getClue();
-    return ( 
-      <h3>{clue.question}</h3> 
-      );
-  }
-  
-  const renderHint = () => {
-    const clue = getClue();
-    return (
-      <strong>{clue.Hint}</strong>
-      );
-  }
-
-  const handleSubmit = (event) => {
-    event.preventDefault();
-    console.log("Answer Submitted is:", answer);
-  }
-
-  const handleChange = (event) => {
-    setAnswer(event.target.value);
+    return Clues[index]
   }
 
   return (
-    <Grid container spacing={3}>
+    <Grid container spacing={0} alignItems="stretch">
       <Grid item xs={2} />
```

Comment:
I dont think we need ``` <Grid item xs={2} />``` for both side. Use this instead 
``` 
<Grid container spacing={0} justify="center" alignItems="stretch">
  <Grid item xs={8}>
   <QuestionOne />
   <QuestionTwo />
   <QuestionThree />
 </Grid>
<Grid>
```

Reply:
Actually we will need it, since we want to make sure content do not overwrite on those two arrows

Reply:
Basically, it works the same way using ```justify="center"``` will center the grid which has right and left offset by 2.

Reply:
Huh, today I learn. Thanks